src/main/java/com/conveyal/gtfs/error/DuplicateStopError.java

```diff
@@ -11,9 +11,9 @@
     private final String message;
     public final DuplicateStops duplicateStop;
 
-    public DuplicateStopError(String message, DuplicateStops duplicateStop) {
-        super("stop", 0, "stop_lat,stop_lon");
-        this.message = message;
+    public DuplicateStopError(DuplicateStops duplicateStop) {
+        super("stop", duplicateStop.stop1.sourceFileLine, "stop_lat,stop_lon", duplicateStop.getStop1Id());
```

Comment:
@mattwigway Is there a good shorthand way to do this in the `super` call (without a long, ugly ternary statement)?